book/src/00_introduction.md

```diff
@@ -1,8 +1,5 @@
 # Introduction
 
-> Note: EDR and this book are currently under active development.
```

Comment:
To make reviews of markdown simpler, we're purposefully putting one sentence per line. When entire paragraphs are combined into a single line, it blows up the diffs (as happened here).

Could you please revert the formatter commit?

Reply:
Done.